mrmustard/physics/bargmann.py

```diff
@@ -94,14 +91,14 @@ def wigner_to_bargmann_Choi(X, Y, d):
         [[I, 1j * I, o, o], [o, o, I, -1j * I], [I, -1j * I, o, o], [o, o, I, 1j * I]]
     ) / np.sqrt(2)
     A = math.matmul(math.matmul(R, A), math.dagger(R))
-    A = math.matmul(A, math.Xmat(2 * N))  # yes: X on the right
+    A = math.matmul(math.Xmat(2 * N), A)
     b = math.matvec(xi_inv, d)
     B = math.matvec(math.conj(R), math.concat([b, -math.matvec(XT, b)], axis=-1)) / math.sqrt(
         settings.HBAR, dtype=R.dtype
     )
-    B = math.concat([B[2 * N :], B[: 2 * N]], axis=-1)  # yes: opposite order
+    B = math.asnumpy(B)
```

Comment:
why convert to numpy?

Reply:
I tested in a notebook with this part of code and I've got a list here not an array, so I'm afraid the type of this vector would be wrong and I just add one line to ensure it...

Reply:
but then we lose differentiability no?

Reply:
You are right, I was thinking to use the function in math would keep the differentiability... It would be better just delete this line.